docs/getting-started-guides/docker-multinode/master.sh

```diff
@@ -75,6 +75,7 @@ detect_lsb() {
     fi
 
     lsb_dist="$(echo ${lsb_dist} | tr '[:upper:]' '[:lower:]')"
+    echo "Detected $lsb_dist"
```

Comment:
should be fixed but the bug was not introduced in this PR so we can fix in a followup